src/arrayMethodSort.js

```diff
@@ -3,9 +3,33 @@
 /**
  * Implement method Sort
  */
+function compareStrings(a, b) {
+  const strA = a.toString();
+  const strB = b.toString();
+
+  return strA > strB ? 1 : strA < strB ? -1 : 0;
```

Comment:
I suppose if we have more than two occasions it better to use if else condition.